default_kms_key_arn.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python2
+import sys
+import json
+import boto3
+
+# setup connection to AWS
+client = boto3.client('kms')
+
+# get the default KMS key
+key_aliases = client.list_aliases()['Aliases']
+
+arn = None
+
+for alias in key_aliases:
+    if alias['AliasName'] == 'alias/aws/lambda':
+        arn = alias['AliasArn']
+
+if not arn:
+    json.dump({
+        'default_lambda_kms_key_arn': ""
+    }, sys.stdout, indent=2)
+    sys.stdout.write('\n')
+    print("Aborting could not find default KMS key for Lambda: "
```

Comment:
I think `sys.stderr.write` would be better than `print` here. It would need a trailing `\n` too.

Also, please add a comma after `Aborting`.

Reply:
Switched to sys.stderr.write and added the comma as requested.